.github/workflows/stale-issue-cleanup.yml

```
@@ -0,0 +1,53 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  # allows doing a manual trigger of the action
+  workflow_dispatch: 
+  # disable cron until dry run is completed successfully
+  # schedule:
+  # - cron: "0 0 * * *"
+
+permissions: {}
+jobs:
+  cleanup:
+    permissions:
+      issues: write # to label, comment and close issues (aws-actions/stale-issue-cleanup)
+     
+    runs-on: ubuntu-latest
+    name: Stale issue job
```

Never needed to mention "job" in a job name. Mentioning this to avoid proliferation of this pattern. It seems to pop up in lots of our infrastructure.

```suggestion
    name: stale issues
```

---

Updated